ledger/test-common/BUILD.bazel

```diff
@@ -141,6 +141,7 @@ EOF
             generated_srcs = [":dar-file-names-scala-%s" % target],
             # generated_srcs is required for scaladoc
             resources = ["dar-files-%s" % target],
+            tags = ["maven_coordinates=com.daml:test-dar-files-%s-lib:__VERSION__" % target],
```

Comment:
If you want to publish those artifacts, you should problably be more 


             for target in lf_versions_aggregate("legacy", "stable", "latest", "preview", "dev", "1.8", "1.11", "1.12")
```